common/src/main/java/gov/cms/ab2d/common/util/Constants.java

```diff
@@ -70,10 +70,15 @@ private Constants() { }
 
     public static final String ZIP_SUPPORT_ON = "ZipSupportOn";
 
+    public static final String MAX_DOWNLOADS = "file.download.count.max";
+
+    public static final String RE_DOWNLOAD_MAX_INTERVAL_MINUTES = "file.download.max.interval.minutes";
+
```

Comment:
Do these really need to be made properties?

More of a philosophical discussion, perhaps but in my experience in multiple companies is that the tendency is to make way too much stuff a property.  Properties that never are changed have more cost than value.  Given it is not too hard to change a constant and do a deploy, I make the case that any new run time configurable properties justify their existence.

I vote for simply making these constants.  If we find that we actually need to change the policy on a regular basis, then they could be promoted to properties.